plugins/catalog-backend-module-aws/src/providers/AwsS3EntityProvider.ts

```diff
@@ -62,7 +62,11 @@ export class AwsS3EntityProvider implements EntityProvider {
     // and the default integration will be added as second integration.
     // In this case, we still want the first one though, but have no means to select it
     // just from the bucket name (and region).
-    const integration = ScmIntegrations.fromConfig(configRoot).awsS3.list()[0];
+    const integrations = ScmIntegrations.fromConfig(configRoot).awsS3.list();
+    if (!integrations) {
```

Comment:
ah but the list is never null, i meant that it'd be of zero length so that `[0]` returns undefined

Reply:
fixed